gallery/src/components/NxGlobalSidebarFooter/NxGlobalSidebarFooterMinimalExample.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import {
+  NxGlobalSidebar,
+  NxGlobalSidebarNavigation,
+  NxGlobalSidebarNavigationLink,
+  NxGlobalSidebarFooter,
+  useToggle,
+  NxPageMain }
+  from '@sonatype/react-shared-components';
+import { faArrowLeft, faArrowRight, faLink, faExternalLinkSquareAlt } from '@fortawesome/free-solid-svg-icons';
+
+const logoImg = require('../../assets/images/logo-plaid-villain-text.png');
+
+export default function NxGlobalSidebarFooterExample() {
+  const [sidebarOpen, onToggleCollapse] = useToggle(true);
+
+  return (
+    <>
+      <NxGlobalSidebar isOpen={sidebarOpen}
+                       toggleOpenIcon={faArrowLeft}
+                       toggleCloseIcon={faArrowRight}
+                       onToggleClick={onToggleCollapse}
+                       logoImg={logoImg}
+                       logoAltText="RSC Plaid Villain"
+                       logoLink="#">
+        <NxGlobalSidebarNavigation>
```

Comment:
IMO all of the footer examples should use identical content outside of the footer, to draw attention to the intended differences between the examples

Reply:
> IMO all of the footer examples should use identical content outside of the footer, to draw attention to the intended differences between the examples

https://github.com/sonatype/sonatype-react-shared-components/pull/387/commits/74b60f8e85c31534b0a13344944cb02698498850